_posts/2022-12-05-solidity-core-team-updates.md

```diff
@@ -0,0 +1,142 @@
+
+
+### Nikola - @[nikola-matic](https://github.com/nikola-matic)
+
+#### What is your professional background?
+
+_I have a BSc in Comp Sci, and a ~10 year long obsession with C++, which kinda narrowed down my options as far as jobs were concerned, because no one's really gonna write a web app in C++. Having said that, I ended up working on a ton of interesting projects throughout my career, like writings high-perf server-client applications for SMS messaging, hardware virtualization on the cloud, EV routing, and a ton of other stuff. Unfortunately, up until I joined the EF and the Solidity team, I've never worked on an open source project, so this is a first for me (hopefully not the last)._
+
+#### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+
+_I've only joined the team ~4 months prior to writing this, so not that much. I did implement the option for the user to pass the clean-up steps for the optimizer sequence (both CLI and JSON), so at least that's something. At the moment, and for the foreseeable future, I'll be working on helping Chris (chriseth) with speeding up the `--via-ir` pipeline, as well as getting the experimental version of the standard library merged._
+
+#### What do you think is Solidity’s biggest challenge?
+
+_Fundamentally, I think that we as the Solidity team don't use the language enough, which is especially concerning seeing as we design and implement it. This is something that the community has pointed out in the past as well, and is something we're well aware of. Thankfully, we are working on this, and plan to have internal team projects and hackathons in the future, which should hopefully give us a different perspective going forward._
+
+
+### Nuno - @[NunoFilipeSantos](https://github.com/NunoFilipeSantos)
+
+#### What is your professional background?
+
+_I have a MSc in Software Engineering and all my life I have been highly motivated to work with open-source software. In recent years, I have worked for SUSE, the largest independent open-source company in the world, and Timescale, an open-source time-series database extension for PostgreSQL as a Product Manager. I am usually also an informal Agile coach for most of my teams with a heavy focus on execution and team happiness/motivation. I have been a remote worker for almost 10 years now and have been 100% remote since 2018._
+
+#### What would you like to see in the Solidity ecosystem in the future?
+
+_Solidity will start having a more community-friendly roadmap and I would love it if the community could engage with us on those issues, especially in our forum since they can positively influence our plans. Another personal motivation is improving developer ergonomics with better debugging tools or documentation._
```

Comment:
```suggestion
_Solidity will start having a more community-friendly [roadmap](https://github.com/orgs/ethereum/projects/26/views/1), and I would love it if the community could engage with us on those issues, especially in our [forum](https://forum.soliditylang.org/), since they can positively influence our plans. Another personal motivation is improving developer ergonomics with better-debugging tools or documentation._
```